FlightReservation/components/signupView.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Text, 
+} from 'react-native';
+import ModalCustom from './modalComponent';
+import SignUpButton from './signUpButtons';
+import Form from './formComponent';
+import Terms from './termsComponents';
+
+
+
+function SignupForm(props) {
+  const {setIsLoginFormActive,isLoginFormActive,showObj, showModal, addFill, formObjectState, modalVisible, singedText, isIconCheck} = props;
```

Comment:
this line is horizontally really long, split it like this:
```suggestion
  const {
    setIsLoginFormActive,
    isLoginFormActive,
    showObj,
    showModal,
    addFill,
    formObjectState,
    modalVisible,
    singedText,
    isIconCheck,
    } = props;
```